device-types/Ciena/8140.yaml

```diff
@@ -0,0 +1,162 @@
+---
+manufacturer: Ciena
+model: '8140'
+slug: ciena-8140
+part_number: 170-8140-900
+u_height: 2
+is_full_depth: false
+weight: 17
+weight_unit: kg
+subdevice_role: parent
```

Comment:
Generally you do not need `subdevice_role` unless the device either slots into another (in which case it would be a child), or has `device-bays` to house child devices. If this device does not need this, please remove it